tailwind.config.js

```diff
@@ -10,7 +10,7 @@ export default {
   theme: {
     extend: {},
     fontFamily: {
-      lora: ['Lora', 'serif']
+      lora: ['Lora', 'serif'],
```

Comment:
put intr font families here. then you can use font-intr

Reply:
done